package.xml

```diff
@@ -21,16 +21,23 @@
   <author>Dave Hershberger</author>
   <author>Dorian Scholz</author>
   <author>Dirk Thomas</author>
+  <author>Stephen Brawner</author>
 
-  <buildtool_depend>catkin</buildtool_depend>
+  <buildtool_depend>ament_cmake</buildtool_depend>
 
-  <build_depend>rosbuild</build_depend>
   <build_depend>qtbase5-dev</build_depend>
-  <build_depend>python-qt5-bindings</build_depend>
+  <build_depend>python3-qt5-bindings</build_depend>
 
-  <run_depend>python-qt5-bindings</run_depend>
+  <exec_depend>python3-qt5-bindings</exec_depend>
+
+  <test_depend>ament_cmake_pytest</test_depend>
+  <test_depend>ament_lint_auto</test_depend>
+  <test_depend>ament_flake8</test_depend>
+  <test_depend>ament_pep257</test_depend>
+  <test_depend>python3-pytest</test_depend>
+  <test_depend>python3-qt5-bindings</test_depend>
```

Comment:
Since this is already an `exec_depend` it doesn't need to be declared as a `test_depend`.